### PR TITLE
inbox list for teams - mobile

### PR DESCRIPTION
**Pull request by chrisnojima**
Mobile version of inbox list

### CODE REVIEW

File: shared/chat/inbox/container.js [hunk 1]
```diff
@@ -54,7 +53,9 @@ const filteredInbox = createImmutableEqualSelector(
           if (!bigTeamToChannels[i.teamname]) {
             bigTeamToChannels[i.teamname] = {}
           }
-          bigTeamToChannels[i.teamname][i.channelname] = value
+          // Do we have the real name yet?
+          const channel = i.channelname === '-' ? id : i.channelname
```
Comment (chrisnojima) [inline shared/chat/inbox/container.js]: else they all have the same name and the map collapses

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/actions/chat/inbox.js [hunk 2]
```diff
@@ -131,7 +131,7 @@ const onInboxStale = function*(): SagaGenerator<any, any> {
       (inbox.items || [])
         .map(c => {
           return new Constants.InboxStateRecord({
-            channelname: c.membersType === ChatTypes.CommonConversationMembersType.team ? ' ' : undefined,
+            channelname: c.membersType === ChatTypes.CommonConversationMembersType.team ? '-' : undefined,
```
Comment (MarcoPolo) [inline shared/actions/chat/inbox.js]: I don't understand how this makes sense

**Comment by MarcoPolo** [comment]
The storybook version on native doesn't use the full screen

File: shared/actions/chat/inbox.js (hunk 2 above)
Comment (chrisnojima) [inline shared/actions/chat/inbox.js]: we don't have the team name yet as its not part of the untrusted call, yet its still a channel so we need a key but dont' know the value

**Pull request merged**